### DISCUSSION TITLE
Rate Limits for Github App REST APIs invoked on the App

### DISCUSSION THREAD

**Discussion by nikhilbirmiwal** [community/community#46684] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hello all!

I'm working with a Github app and have some questions about their rate limits.

For Github Apps that are installed (e.g. repo, organization), I see that the rate limits are [~15,000](https://docs.github.com/en/developers/apps/building-github-apps/rate-limits-for-github-apps). Further, I know that we can get the rate limit remaining by looking at the response headers (e.g. `x-ratelimit-remaining`).

However, some Github App APIs aren't attached to an installation; instead, they provide [information about installations](https://docs.github.com/en/rest/apps?apiVersion=2022-11-28). I'm curious what the rate limits are for these APIs.

Locally experimenting with [this endpoint](https://docs.github.com/en/rest/apps/webhooks?apiVersion=2022-11-28#list-deliveries-for-an-app-webhook), I see some documentation about being cognizant of error code 422. I also didn't see any `x-ratelimit-remaining` responses in the headers.

I'm curious what the rate limits are for APIs that are aren't attached to installations.

Best,
Nikhil

**Comment by Anmol-Baranwal** [comment, marked as answer]
The rate limits for APIs that are not attached to installations on GitHub may vary depending on the specific API and the endpoint being used. It's possible that some of these APIs have different rate limits or don't have any rate limits at all.

You mentioned an API endpoint that provides information about installations, but doesn't seem to have any rate limit headers in its responses. In such cases, it's possible that the API doesn't have rate limits or that the headers are not being returned due to a bug or issue with the endpoint.
